muspinsim/muspinsim.xml

```diff
@@ -0,0 +1,65 @@
+<tool id="muspinsim" name="Simulate muon experiment spin dynamics (muspinsim)" version="0.1.0" python_template_version="3.5">
+    <requirements>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        pip install git+https://github.com/muon-spectroscopy-computational-project/muspinsim.git &&
+        ln -s $mu_sim $mu_sim.name &&
+        ln -s $mu_exp_in $mu_exp_in.name &&
```

Comment:
If `$x.name` contains spaces (as is possible within the Galaxy interface), the symlinking doesn't work properly - see https://github.com/muon-spectroscopy-computational-project/muon-galaxy-tools/issues/54 . We should use a different name here. 
A hard-coded name is easiest, though there's also the option to replace spaces in the input name with e.g. underscores, which has benefits if the file name will be written in any logs. It's nice for users to be able to see file names they recognize throughout.
Bear in mind that this change also affects other lines in this script!

Reply:
So i fixed this issue by creating two new variables to hold `mu_sim.name` and `mu_exp_in.name` and removing any spaces found. That way we can keep the names similar in the logs